index.js

```diff
@@ -14,11 +17,21 @@ module.exports = {
     this._ensureThisImport();
 
     this.import('vendor/ember-simple-auth/register-version.js');
+    this.import('vendor/base64.js');
   },
 
-  treeForVendor: function() {
-    var content = 'Ember.libraries.register(\'Ember Simple Auth\', \'' + version + '\');';
-    return writeFile('ember-simple-auth/register-version.js', content);
+  treeForVendor() {
+    var content =
+      "Ember.libraries.register('Ember Simple Auth', '" + version + "');";
+    var registerVersionTree = writeFile(
+      'ember-simple-auth/register-version.js',
+      content
+    );
+    var base64Tree = new Funnel(path.dirname(require.resolve('base-64')), {
+      files: ['base64.js']
+    });
```

Comment:
I think this will not automatically polyfill the `atob` and `btoa` methods. We'd probably have to do that outselves in a vendor file.